src/wp-includes/script-loader.php

```diff
@@ -231,81 +231,39 @@ function wp_get_script_polyfill( &$scripts, $tests ) {
 function wp_default_packages_scripts( &$scripts ) {
 	$suffix = wp_scripts_get_suffix();
 
-	$packages = array(
-		'a11y',
-		'annotations',
-		'api-fetch',
-		'autop',
-		'blob',
-		'block-directory',
-		'block-editor',
-		'block-library',
-		'block-serialization-default-parser',
-		'blocks',
-		'components',
-		'compose',
-		'core-data',
-		'data',
-		'data-controls',
-		'date',
-		'deprecated',
-		'dom',
-		'dom-ready',
-		'edit-post',
-		'editor',
-		'element',
-		'escape-html',
-		'format-library',
-		'hooks',
-		'html-entities',
-		'i18n',
-		'is-shallow-equal',
-		'keyboard-shortcuts',
-		'keycodes',
-		'list-reusable-blocks',
-		'media-utils',
-		'notices',
-		'nux',
-		'plugins',
-		'primitives',
-		'priority-queue',
-		'redux-routine',
-		'rich-text',
-		'shortcode',
-		'server-side-render',
-		'token-list',
-		'url',
-		'viewport',
-		'warning',
-		'wordcount',
-	);
+	// Expects multidimensional array like:
+	//	'a11y.js' => array('dependencies' => array(...), 'version' => '...'),
+	//	'annotations.js' => array('dependencies' => array(...), 'version' => '...'),
+	//	'api-fetch.js' => array(...
+	$assets = include ABSPATH . WPINC . '/assets/script-loader-data.php';
```

Comment:
I know it might be bikesheeding, but `script-loader-package-assets.php` or `script-loader-packages.php` seems to be a better name since the file will only contain packages and not any other generic data for the script loader.

Reply:
Sure, sounds good (hardest part of coding -- coming up with good names) :)